x-pack/plugins/serverless_search/public/application/components/languages/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum Languages {
```

Comment:
do we want to update this as serverless client languages are created? since it will be a subset of languages elasticsearch client supports at first.

Reply:
I'm tracking which ones are enabled with the languages array in `languages.ts`.